spec/requests/odata/metadata_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe "$metadata" do
+  include_context "odata"
+
+  let(:path) { "#{mission_api_route}/$metadata" }
+
+  context "with no forms" do
+    it "renders as expected" do
+      expect_output_fixture("empty_metadata.xml")
+    end
+  end
+
+  context "with several basic forms" do
+    include_context "odata_with_basic_forms"
+
+    it "renders as expected" do
+      expect_output_fixture("basic_metadata.xml", forms: [form, form_with_no_responses])
+    end
```

Comment:
You can just do `it { expect_output_fixture(...) }` without loss of clarity. I will typically get rid of the blank line above `it` in that case too. Which makes things nice and compact.